src/devices/GrovePi/samples/Program.cs

```diff
@@ -4,45 +4,53 @@
 
 using System;
 using System.Threading;
+using System.Device.I2c;
+using System.Device.Gpio;
 using Iot.Device.GrovePiDevice.Models;
 using Iot.Device.GrovePiDevice;
 using Iot.Device.GrovePiDevice.Sensors;
-using System.Device.I2c;
-using System.Device.Gpio;
 
 namespace GrovePisample
 {
-    class Program
+    /// <summary>
+    /// Test class for the GrovePi
+    /// </summary>
+    public class Program
     {
-        static private GrovePi grovePi;
-        static void Main(string[] args)
+        private static GrovePi _grovePi;
```

Comment:
It doesn't matter much here given that this is just a sample, but technically our coding guidelines say that private static fields should be named like: `s_grovePi`. I wonder if we could add that as one of the rules

Reply:
I think that only works with a custom rule, at least I can't find one for that requirement. SX1309S makes static fields require underscores, but otherwise there doesn't seem to be any rules explicitly for static members (there are some for static readonly members, but these we handle like constants now, so they use PascalCase)

Reply:
I see that is fine, we can always write a new roslyn rule in the future.